common/Util.hpp

```diff
@@ -206,8 +206,7 @@ namespace Util
     }
 
     /// Hex to unsigned char
-    template <typename T>
-    bool dataFromHexString(const std::string& hexString, T& data)
+    template <typename T> bool dataFromHexString(const std::string_view hexString, T& data)
```

Comment:
You mean `const&` instead of passing by-value?

The goal is to have `string_view` in most all of the parsing path. I have more similar patches. I'd like us to use `string_view` in most of these functions.

And it turns out that if you already have a `string_view`, passing it by-value is actually faster than by-ref, at least on x86-64. That's because the components of `string_view` are passed via registers and two indirections are avoided (one to write the reference to the stack, and one to read it in the callee). The parsers are very heavily used, of course.

Granted, the function above is used mostly for debugging, so this point is moot. Still, I went for consistency here, rather than special-casing.